### PR TITLE
GH-117 Adds short code generate button tooltip

### PR DESCRIPTION
**Pull request by milindmore22**
related #117

### CODE REVIEW

File: templates/backend/contribution/create.php [hunk 1]
```diff
@@ -59,9 +59,14 @@
...
 				</label>
-				<button class="rev-gen-contribution-main-generate-button">
-					<?php esc_html_e( 'Generate and copy code', 'revenue-generator' ); ?>
-				</button>
+				<div id="rg_contribution_generate" class="rev-gen-contribution-main-button">
+					<button class="rev-gen-contribution-main-generate-button">
+						<?php esc_html_e( 'Generate and copy code', 'revenue-generator' ); ?>
+					</button>
+					<a target="_blank" href="https://wordpress.com/support/shortcodes/" id="rev-gen-contribution-help-generate" class="rev-gen-contribution-main--help-button">
+							<img src="<?php echo esc_url( $action_icons['option_info'] ); ?>">
```
Comment (mangesh) [inline templates/backend/contribution/create.php:67]: Can we remove this extra indentation?
```suggestion
						<img src="<?php echo esc_url( $action_icons['option_info'] ); ?>">
```

File: templates/frontend/contribution-dialog.php [hunk 2]
```diff
@@ -63,7 +63,8 @@
 						<div class="rg-custom-amount-goback">
 							<img src="<?php echo esc_url( $action_icons['back_arrow_icon'] ); ?>" class="arrow" />
 						</div>
-						<input class="rg-custom-amount-input" id="rg-custom-tip-amount" name="tip" type="number" value="" step="0.01" min="0" placeholder="$20.00" onfocus="this.placeholder = ''; this.value = (this.value?this.value:20);" onblur="this.placeholder = '$20.00'">
+						<?php $default_amount = $currency_symbol . '20'; ?>
+						<input class="rg-custom-amount-input" id="rg-custom-tip-amount" name="tip" type="number" value="" step="0.01" min="0" placeholder="<?php echo esc_attr( $default_amount ); ?>" onfocus="this.placeholder = ''; this.value = (this.value?this.value:20);" onblur="this.placeholder = '<?php echo esc_attr( $default_amount ); ?>'">
```
Comment (mangesh) [inline templates/frontend/contribution-dialog.php:67]: Can we increase some security here by applying `wp_json_encode()` on `$currency_symbol ` and passing the 
`$default_amount` in `esc_attr()`? You can refer this following example.
https://developer.wordpress.org/reference/functions/esc_js/#comment-1747

PS: If we follow this method we will need some testing as well.

Comment (milindmore22) [reply templates/frontend/contribution-dialog.php]: Feedback Addressed, Please review again

**Comment by milindmore22** [comment]
@mangesh  Feedback Addressed please review again.

**Pull request merged**